Topics/User_Experience/User_Research.md

```diff
@@ -0,0 +1,102 @@
+## Resources for User Research domain
+
+### Overview
+
+User research is an essential part of the user experience (UX) design process. It involves gathering information about users and their needs, behaviors, and attitudes through various research methods. The goal of user research is to create a user-centered design by understanding the needs and goals of the users and incorporating this information into the design process. the key topics in user research include User Personas, User Testing, Surveys and Questionnaires, Interviews, and Contextual Inquiry.
+
+User research can be conducted through various methods, such as surveys, interviews, focus groups, usability testing, and ethnographic studies. These methods allow researchers to gather information about users' experiences, behaviors, and preferences, which can then be used to inform design decisions.
+
+### Key Topics in User Research
+
+1. User Personas: User personas are fictional representations of your target users. They help you understand the needs, goals, and behaviors of your users, which can inform the design of your product. Creating user personas involves conducting research to gather information about your users, such as demographic data, behavioral patterns, and attitudes.
```

Comment:
The below paragraphs are subparts of this paragraph, explaining how to create user personas, so the indent should be larger than this paragraph for a better reading experience. Also, I suggest making each topic bold (in this line "User Personas: ") so the reader can find the topic they want to learn easier.